slave-base/contrib/bin/run-jnlp-client

```diff
@@ -65,57 +63,88 @@ if [[ $# -gt 1 ]]; then
     fi
   fi
 
-  CONTAINER_MEMORY_IN_BYTES=`cat /sys/fs/cgroup/memory/memory.limit_in_bytes`
-  DEFAULT_MEMORY_CEILING=$((2**40-1))
-  if [ "${CONTAINER_MEMORY_IN_BYTES}" -lt "${DEFAULT_MEMORY_CEILING}" ]; then
+  if [[ -z "${JAVA_TOOL_OPTIONS}" ]]; then
+    # these options will automatically be picked up by any child JVM process
```

Comment:
any child jvm process or just "any jvm process"?

Reply:
will change to "any JVM process".  I mean(t) the spawned JNLP JVM and any subsequent JVMs it may spawn.

Reply:
just for my education, JAVA_OPTS was a first class env var of sorts, specifically looked for by say maven and/or gradle or some such, and hence we are avoiding overlap, right?